cmd/metal-api/internal/service/project-service.go

```diff
@@ -25,7 +25,11 @@ type projectResource struct {
 }
 
 // NewProject returns a webservice for project specific endpoints.
-func NewProject(log *zap.SugaredLogger, ds *datastore.RethinkStore, mdc mdm.Client) *restful.WebService {
+func NewProject(
```

Comment:
Nothing really changed, also we do not use C-style function declaration style so much... :D